dnf-json

```diff
@@ -82,6 +82,29 @@ class Solver():
         if "sslclientcert" in desc:
             repo.sslclientcert = desc["sslclientcert"]
 
+        if "check_gpg" in desc:
+            repo.gpgcheck = desc["check_gpg"]
+        if "check_repogpg" in desc:
+            repo.repo_gpgcheck = desc["check_repogpg"]
+        if "gpgkey" in desc:
```

Comment:
Is this kept for compatibility reasons?

Reply:
correct, no need to break dnf-json's api.